docs/using_eessi/setting_up_environment.md

```diff
@@ -1,6 +1,48 @@
 # Setting up your environment
 
-To set up the EESSI environment, simply run the command:
+To set up the EESSI environment, you can either load an EESSI module with Lmod or source a bash initialisation script.
+
+## Loading an EESSI module with Lmod
+
+There are a few different scenarios where you may want to set up the EESSI environment by loading an EESSI module with Lmod:
+
+1.  You are already using Lmod with version >= 8.6
+
+    In this case, we recommend unsetting the `MODULEPATH`, because EESSI is not designed to mix modules coming from EESSI and from your system.
+    The underlying reason is that very likely EESSI and your system are based on two different operating system distributions - EESSI uses Gentoo, your system likely uses some other distribution. Note, unsetting `MODULEPATH` may not be possible at some sites, and some compromise may be necessary:
+
+    ``` { .bash .copy }
+    unset MODULEPATH
+    export MODULEPATH=/cvmfs/software.eessi.io/init/modules
+    module load EESSI/2023.06
+    ```
+
```

Comment:
```suggestion
    !!! note "Why should I unset `MODULEPATH`?"
        Unsetting `MODULEPATH` may not be possible at some sites, and some compromise or workaround may
        be necessary. The underlying reason to suggest this is that EESSI and your system are most likely based on
        two different operating system distributions - EESSI uses Gentoo Prefix, your system almost certainly uses some other
        distribution. If you can find a way to ensure that the _software stacks_ from your site and EESSI do not mix
        (in particular when someone is building new software!), then this should be good enough.

```

Reply:
It may be better to move this up entirely as the reasoning is value for all items

Reply:
> It may be better to move this up entirely as the reasoning is value for all items

Moved it upwards